identityClient/identityClient.go

```diff
@@ -516,6 +517,16 @@ func (c *E3dbIdentityClient) InternalDeleteIdentity(ctx context.Context, realmNa
 	return err
 }
 
+func (c *E3dbIdentityClient) InternalDeleteIdentitiesByProvider(ctx context.Context, realmName string, providerID uuid.UUID) error {
```

Comment:
prefer adding a struct vs an additional param, if only for OCD/minalimiim of having to not import the uuid library :-P. Clean and lean

Reply:
I'm lodging a formal protest because I think the uuid library should be imported everywhere just in case you might need it, it is a travesty it isn't just baked into the standard library 😛. but ok